Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-slim
+
+ARG FLEET_VERSION=compiler-fleet-1.0-SNAPSHOT
```

Comment:
Its default name (if you make git clone, gradl distTar) is `papeeria-compiler-fleet-1.0-SNAPSHOT.tar`. Please change this so that it could work with default values.

Reply:
Done